AUTHORS.txt

```diff
@@ -18,6 +18,7 @@ The following developers contributed to gcovr (ordered alphabetically):
     Christian Taedcke,
     Dave George,
     Dom Postorivo,
+    Elektrobit Automotive GmbH,
```

Comment:
Do you have the aggrement of your legal department to contribute in the name of the company?

Reply:
Yes. In fact, I am not allowed to use my own name for this, since I do the code changes on company time. Do you need this in written form?

Reply:
@latk This is the first time a company is added to the Authors file. What do you think about this?